projects/1.21/assets/rar-compat/rarcompat/lang/zh_cn.json

```
@@ -0,0 +1,95 @@
+  "tooltip.relics.plastic_drinking_hat.ability.drinking.stat.speed.value": "+%1$s%%。",
+  "tooltip.relics.plastic_drinking_hat.ability.nutrition": "饱腹啜饮",
+  "tooltip.relics.plastic_drinking_hat.ability.nutrition.description": "饮用液体的同时恢复玩家的饥饿值。",
+  "tooltip.relics.plastic_drinking_hat.ability.nutrition.stat.hunger.title": "恢复的饥饿值：",
+  "tooltip.relics.plastic_drinking_hat.ability.nutrition.stat.hunger.value": "%1$s点。",
+
+  "tooltip.relics.novelty_drinking_hat.description": "加速饮用速度，不只能迅速解渴，甚至还能恢复饥饿值。",
+  "tooltip.relics.novelty_drinking_hat.ability.drinking": "迅速饮用",
+  "tooltip.relics.novelty_drinking_hat.ability.drinking.description": "增加饮用速度。",
+  "tooltip.relics.novelty_drinking_hat.ability.drinking.stat.speed.title": "额外饮用速度：",
+  "tooltip.relics.novelty_drinking_hat.ability.drinking.stat.speed.value": "+%1$s%%。",
+  "tooltip.relics.novelty_drinking_hat.ability.nutrition": "饱腹啜饮",
+  "tooltip.relics.novelty_drinking_hat.ability.nutrition.description": "饮用液体的同时恢复玩家的饥饿值。",
+  "tooltip.relics.novelty_drinking_hat.ability.nutrition.stat.hunger.title": "恢复的饥饿值：",
+  "tooltip.relics.novelty_drinking_hat.ability.nutrition.stat.hunger.value": "%1$s点。",
+
+  "tooltip.relics.snorkel.description": "能让你长时间在水下探索，而不必担心溺水。",
+  "tooltip.relics.snorkel.ability.diving": "呼吸控制",
+  "tooltip.relics.snorkel.ability.diving.description": "完全浸入水中后给予持续一段时间的水下呼吸效果。",
+  "tooltip.relics.snorkel.ability.diving.stat.duration.title": "效果持续时间：",
+  "tooltip.relics.snorkel.ability.diving.stat.duration.value": "%1$s秒。",
+
+  "tooltip.relics.villager_hat.description": "降低与村民交易的价格，让贸易更有利可图。",
+  "tooltip.relics.villager_hat.ability.discount": "贸易之秘",
+  "tooltip.relics.villager_hat.ability.discount.description": "与村民交易时增加折扣量。",
+  "tooltip.relics.villager_hat.ability.discount.stat.multiplier.title": "折扣量：",
+  "tooltip.relics.villager_hat.ability.discount.stat.multiplier.value": "%1$s%%",
+
+  "tooltip.relics.superstitious_hat.description": "增加被击杀生物的战利品掉落。",
+  "tooltip.relics.superstitious_hat.ability.looting": "大批战利品",
+  "tooltip.relics.superstitious_hat.ability.looting.description": "有概率在击杀生物时附带一级抢夺。若触发成功，则会再次尝试触发以再附带一级抢夺。此流程会持续到概率判定失败，并将所有等级加和计算。",
+  "tooltip.relics.superstitious_hat.ability.looting.stat.chance.title": "触发概率：",
+  "tooltip.relics.superstitious_hat.ability.looting.stat.chance.value": "%1$s%%",
+
+  "tooltip.relics.anglers_hat.description": "增加垂钓获得的渔获量。",
+  "tooltip.relics.anglers_hat.ability.catch": "丰足渔获",
+  "tooltip.relics.anglers_hat.ability.catch.description": "有概率增加渔获的数量。若触发成功，则会再次尝试触发以再增加渔获量。此流程会持续到概率判定失败，并将所有增加量加和计算。",
+  "tooltip.relics.anglers_hat.ability.catch.stat.chance.title": "触发概率：",
+  "tooltip.relics.anglers_hat.ability.catch.stat.chance.value": "%1$s%%",
+
+  "tooltip.relics.lucky_scarf.description": "增加破坏方块时的时运。",
+  "tooltip.relics.lucky_scarf.ability.luck": "宝藏猎人",
+  "tooltip.relics.lucky_scarf.ability.luck.description": "有概率在破坏方块时附带一级时运。若触发成功，则会再次尝试触发以再附带一级时运。此流程会持续到概率判定失败，并将所有等级加和计算。",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Has a chance to apply an additional level of luck to mined blocks. If successful, the ability triggers again, adding another level of luck. The process continues until the chance fails, summing all the levels.**

```suggestion
  "tooltip.relics.lucky_scarf.ability.luck.description": "有概率在破坏方块时附带额外一级的时运。若触发成功，则会再次尝试触发以再附带一级时运。此流程会持续到触发失败，之后将所有附加的等级加和计算。",
```

---

等待审完与后续类似句式一起更改